specification.md

```diff
@@ -223,31 +223,31 @@ future, and the rough functionality will be X, Y, Z
 
 ## The Job API; Layer 0
 
-This section describes the most basic form of J/PSI API, namely one in
+This section describes the most basic form of the PSI/J API, namely one in
 which the location of the backend is either implicit or local. The main
 components of the API are shown below:
 
 <img width="100%" src="diagrams/class_diagram.svg" alt="Async Jobs Timing Diagram"/>
 
 The components of the API can be divided into a passive set, describing
 jobs and their state, and an active set, the
-[`JobExecutor`](#job-executor) connectors, which translates job
-information and interacts with specific backends. In other words, the API
+[`JobExecutor`](#jobexecutor) connectors, which translate job
+information and interact with specific backends. In other words, the API
 is designed such that all backend-specific logic can be contained in
 `JobExecutor` connectors. Consequently, all actions, such as `submit()`
 and `cancel()` as well as job status updates belong to the `JobExecutor`
 (although convenience methods also allow the job status to be directly
 retrievable from [`Job`](#job) objects).
 
 The passive set of components has the [`Job`](#job) class at the top. A
-`Job` object has a [`JobSpecification`](#jobspecification), which
+`Job` object has a [`JobSpecification`](#jobspec), which
 describes the high-level details of the job. Here, "high-level" is meant
 to track the capabilities of what can typically be achieved with
 `fork/exec` calls, namely specifying the executable and its arguments,
 the environment of the job, its working directory, and the location of
 input and output streams. Additional information that is specific to
 queuing systems is specified using both the
-[`JobAttributes`](#jobattributes) class, as well as the [`ResourcesSpec`]
+[`JobAttributes`](#jobattributes) class, as well as the `ResourcesSpec`
```

Comment:
It's possible that the link was broken, but it would be nice if ResourceSpec was a link.